docs/contributing/guidelines/workflow.md

```diff
@@ -254,16 +247,26 @@ alpha/beta release etc
 We use many other labels to summarize the scope and effect of the changes:
 
 - *needs: preceding PR* - Cannot yet be merged because it has a dependency on another pull request that needs to merge first.
-- *DO NOT MERGE* - Contains changes that may be in a draft state and submitted purely for review, or may contain breaking changes that have not been considered.
+- *do not merge* - Contains changes that may be in a draft state and submitted purely for review, or may contain breaking changes that have not been considered.
 - *devices: 'name'* - Specifically affects the named device(s).
 - *component: 'name'* - Specifically affects the named component. Component names follow the structure of Mbed OS (for example `ble`, `storage`, `tls`).
 - *rollup PR* - Ready for CI testing, but will instead be brought into a rollup pull request to test multiple pull requests at once.
+- *Release review required* - additional release version approval is required
+- *release version missing* - a merged pull request does not contain release label, it should be fixed by the maintainers team
+- *BREAKING-CHANGE* - a pull request introducing breaking change
+- *mirrored* - the git2jira bot mirrored an issue to our internal Jira
+- *JIRA status:* - an internal jira ticket status propagated to Github issue. It can be: OPEN, IN PROGRESS, CLOSED, RESOLVED or REOPENED.
+- *closed in jira* - mirrored internal Jira issue was closed
+- *Manually patch* - a pull request needs manual cherry-picking for the release candidate to resolve conflicts
+- *rollup PR* - rollup pull request, more details in the section below
+- *risk: x* - How big risks is the pull request for a specific release. Options are: green, amber or red.
 
```

Comment:
Missing: Open for community contribution ?

Reply:
yes, will add. See https://github.com/ARMmbed/mbed-os-5-docs/pull/1222#issuecomment-582910892 . lets discuss these and I'll update the labels here and possibly remove some not used anymore